.gitignore

```diff
@@ -0,0 +1,21 @@
+.DS_Store
```

Comment:
Let’s make `.gitignore` similar to other Logux projects 

https://github.com/logux/core/blob/master/.gitignore